src/box/box.go

```diff
@@ -0,0 +1,33 @@
+package box
+
+import "sync"
+
+//@TODO a collections struct?
+//@TODO what other info can kitty box store?
+//@TODO implement this
+// Box represents a unique kitty box
+type Box struct {
```

Comment:
@128-95 will need to work with you for this :+1: as he is implementing the API for determining the kitty details for a given `KittyID`.

This API will interact with DB where it stores such information, where each information is signed under a public/private key pair.

So it'll be essentially a key-value store of;
* Key: `KittyID`
* Value: 

```go
type SignedBoxType struct {
    BoxType
    Sig cipher.Sig
}
```

But teller will still record what kitties are for sale.

Opinions? :1234:

Reply:
so this will be an external api which will contain the signed kitty data?

Reply:
We will need a boolean in the box to indicates if the box is open or not. Have to think about how we are going to open those boxes.